client/src/test/scala/slack/api/MockSttpBackend.scala

```diff
@@ -0,0 +1,14 @@
+package slack.api
+
+import com.github.dapperware.slack.AccessToken
+import zio.Layer
+import sttp.client3.asynchttpclient.zio.SttpClient
+import sttp.client3.asynchttpclient.zio.SttpClientStubbing
+import sttp.client3.asynchttpclient.zio.AsyncHttpClientZioBackend
+trait MockSttpBackend {
+
+  def sttbBackEndStubLayer: Layer[Nothing, SttpClient with SttpClientStubbing] = AsyncHttpClientZioBackend.stubLayer.mapError(_ => ???)
```

Comment:
Hint: use `.orDie` to submerge the error channel

Reply:
ah, thanks:)